src/test/resources/A&EWatches.feature

```diff
@@ -8,4 +8,6 @@
 #      And I click login amazon button
```

Comment:
After you'll create Amazon steps delete comments

Reply:
I'll change when I'll create Amazon test cases